CHANGELOG.md

```diff
@@ -7,6 +7,11 @@
 - New device: virtio-vsock, backed by Unix domain sockets (GitHub issue #650).
   See `docs/vsock.md`.
 
+### Added
+
+- New parameter, `vmm-config`, which represents a JSON that can be used for configuring 
```

Comment:
nit: Be a bit more specific: "New command-line parameter for both `firecracker` and `jailer`, named `vmm-config`, bla."